python/ray/experimental/state/state_cli.py

```diff
@@ -361,6 +389,18 @@ def summary_state_cli_group(ctx):
 @address_option
 @click.pass_context
 def task_summary(ctx, timeout: float, address: str):
+    """Summarize the task state of the cluster.
```

Comment:
For CLIs, I think it would be nice to always include examples in the docstring, so when people do `--help`, they could use those directly. Like the `ray logs` one below

Reply:
Ah good point. I removed them because it is auto-generated by click, but this is not available when --help is used..